x-pack/plugins/enterprise_search/public/navigation_tree.ts

```diff
@@ -31,7 +31,7 @@ export interface DynamicSideNavItems {
 const title = i18n.translate(
```

Comment:
we could have this use the `SEARCH_PRODUCT_NAME` constant so it always matches classic nav which uses that too.